vm/vm.go

```diff
@@ -79,7 +82,7 @@ type VM struct {
 }
 
 // Validation initializes validation request.
-func (v *VM) Validation(raw types.RawTx) system.ValidationRequest {
+func (v *VM) Validation(raw types.RawTx) system.ValidationRequestNew {
```

Comment:
Why does it return an interface instead of the `Request` struct that implements it? 
```suggestion
func (v *VM) Validation(raw types.RawTx) *Request {
```

It's against the common rule "accept interfaces, return concrete implementations". Is there a need for this?